SurveyApp/Views/AnswerView/MultipleChoiceAnswerView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  MultipleChoiceAnswerView.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/7/21.
+//
+
+import UIKit
+import NimbleExtension
+
+class MultipleChoiceAnswerView: UIView {
+
+    let answers: [SurveyAnswer]
+
+    private let tableView = UITableView()
+
+    init(answers: [SurveyAnswer]) {
+        self.answers = answers
+        super.init(frame: .zero)
+        setUpLayout()
+        setUpViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setUpLayout() {
+        addSubview(tableView)
+        
+        tableView.snp.makeConstraints {
+            $0.centerY.equalToSuperview()
+            $0.leading.equalToSuperview().inset(10.0)
+            $0.trailing.equalToSuperview().inset(20.0)
+            $0.height.equalTo(300.0)
```

Comment:
what will happen to smaller screen? (iPhone SE). I suggest make this height constraint to parent's aspect ratio and make it vertically scrollable (if not already)?